vm.go

```diff
@@ -4539,6 +4672,65 @@ func (_loadNewTarget) exec(vm *vm) {
 	vm.pc++
 }
 
+type _loadImportMeta struct{}
+
+var loadImportMeta _loadImportMeta
+
+func (_loadImportMeta) exec(vm *vm) {
+	// https://262.ecma-international.org/13.0/#sec-meta-properties-runtime-semantics-evaluation
+	t := vm.r.GetActiveScriptOrModule()
+	m := t.(ModuleRecord) // There should be now way for this to have compiled
+	vm.push(vm.r.getImportMetaFor(m))
+	vm.pc++
+}
+
+type _loadDynamicImport struct{}
+
+var dynamicImport _loadDynamicImport
+
+func (_loadDynamicImport) exec(vm *vm) {
+	// https://262.ecma-international.org/13.0/#sec-import-call-runtime-semantics-evaluation
+	vm.push(vm.r.ToValue(func(specifier Value) Value { // TODO remove this function
```

Comment:
```suggestion
	vm.push(vm.r.ToValue(func(specifier Value) Value {
```
Why, when is it expected to happen?

Reply:
Not in this PR - looking at the code I do realize that I did this in order to get this done isntead of writing it to be more optimal. 


We are not going to use dynamic import right now, and it still is fast enough, but it will be nice to fix it in the future.

I have added a note for it.